keybase/platform_windows.go

```diff
@@ -191,12 +199,13 @@ func (c context) Apply(update updater.Update, options updater.UpdateOptions, tmp
 		return fmt.Errorf("No asset")
 	}
 	var args []string
-	auto, _ := c.config.GetUpdateAuto()
-	canBeSilent, _ := CheckCanBeSilent(update.Asset.LocalPath, c.log, CheckRegistryUninstallCode)
-	if auto && canBeSilent {
+	_, auto := c.config.GetUpdateAuto()
+	if auto && !c.config.GetUpdateAutoOverride() {
+
 		args = append(args, "/quiet")
 	}
-	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour, c.log)
+	// We really don't want to kill installers, hence 30 days here
+	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour*24*30, c.log)
```

Comment:
One likely scenario is that the reboot prompt gets killed, then the updater launches the install again, which fails repeatedly because a reboot is required.

You are referring to the case where a bad installer is hung? How to distinguish that from an unanswered dialog?